.github/workflows/ci-tests-pytorch.yml

```diff
@@ -121,6 +121,13 @@ jobs:
           "pytest-timeout" -r requirements/_integrations/accelerators.txt \
           -f ${TORCH_URL} ${TORCH_PREINSTALL} -f ${PYPI_CACHE_DIR} --prefer-binary
         pip list
+    - name: Drop LAI from extensions
+      if: ${{ matrix.pkg-name != 'lightning' }}
+      # Lightning is dependency of Habana or other accelerators/integrations so in case we test PL we need to remove it
```

Comment:
Doing https://github.com/Lightning-AI/lightning-Habana/issues/26 would be better because even though we can do this uninstall in CI, users won't know about it

Reply:
This is not new, just moved to its own step, so we could think about a better solution later...